app/code/community/Bolt/Boltpay/controllers/ShippingController.php

```diff
@@ -62,8 +63,9 @@ public function indexAction()
             $mockTransaction = (object) array("order" => $requestData );
 
             /** @var Bolt_Boltpay_Helper_Transaction $transactionHelper */
-            $transactionHelper = Mage::helper('boltpay/transaction');
-            $quoteId = $transactionHelper->getImmutableQuoteIdFromTransaction($mockTransaction);
+//            $transactionHelper = Mage::helper('boltpay/transaction');
+//            $quoteId = $transactionHelper->getImmutableQuoteIdFromTransaction($mockTransaction);
```

Comment:
Instead of comment out, you should properly update `getImmutableQuoteIdFromTransaction` ?